main.js

```diff
@@ -1 +1,80 @@
-console.log('Hello world')
\ No newline at end of file
+{console.log('Hello world')
+let time, seconds, fullSec, fullMin, fullHour, clicked, turned;

```

Comment:
You aren't using `time`, please remove it!

Also `clicked` and `turned` aren't great variable names. Maybe `timerRunning` instead of clicked?